thirdparty/README.md

```diff
@@ -439,6 +439,20 @@ Files extracted from upstream source:
 - License.txt
 
 
+## Rvo2-3D
+
+- Upstream: http://gamma.cs.unc.edu/RVO2/
+- version: 1.0.1
```

Comment:
Uppercase "Version".

Reply:
I just saw that it's wrong for `recastnavigation` above too, so if you want to fix it at the same time.

Reply:
Done